### PR TITLE
Privacy: link to privacy policy in forms with email opt-in

### PR DESCRIPTION
**Pull request by breville**
### sign-up

screenshot 2018-05-22 14 13 56

### contact a regional partner

screenshot 2018-05-22 17 55 52

### /yourschool survey

screenshot 2018-05-22 14 43 51

### hourofcode.com sign-up

screenshot 2018-05-22 17 56 01

### submit your class at /schools/new

screenshot 2018-05-22 14 59 57

### volunteer

screenshot 2018-05-22 15 25 39

### CODE REVIEW

File: apps/src/code-studio/pd/regional_partner_contact/RegionalPartnerContact.jsx [hunk 1]
```diff
@@ -124,7 +124,18 @@ class RegionalPartnerContactComponent extends FormComponent {
         {
           this.buildButtonsFromOptions({
             name: 'optIn',
-            label: 'Can we email you about updates to our courses, local learning opportunities, or other computer science news?',
+            label: (
```
Comment (breville) [inline apps/src/code-studio/pd/regional_partner_contact/RegionalPartnerContact.jsx:127]: @aoby You might be interested in this particular form.

File: dashboard/app/models/email_preference.rb [hunk 2]
```diff
@@ -15,7 +15,11 @@
 #
 #  index_email_preferences_on_email  (email) UNIQUE
 #
-#
+# Notes:
+#   source: Indicates the source of the opt-in choice.
+#   kind: "0", "1", etc.  Indicates which version of the opt-in string the
+#     user agreed to, for a given source.  This value should be bumped each
+#     time the corresponding user-facing string is updated.
```
Comment (aoby) [inline dashboard/app/models/email_preference.rb]: Thanks for adding these notes!

s/kind/form_kind, so it matches the attribute.

**Comment by tanyaparker** [comment]
Screenshots LGTM

**Comment by breville** [comment]
@Erin007 We should have these strings recorded in source history, so I don't think we need to record them elsewhere.  We could in fact track accepted strings by looking at the timestamp of a submission and matching back to the source tree and the corresponding deploy-to-production time, but having a number which we bump each time felt like a way to make that last part much easier.

**Pull request merged**